staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -105,6 +106,11 @@ func NewPrefixTransformers(err error, transformers ...PrefixTransformer) Transfo
 func (t *prefixTransformers) TransformFromStorage(ctx context.Context, data []byte, dataCtx Context) ([]byte, bool, error) {
 	start := time.Now()
 	var errs []error
+	reqInfo, found := genericapirequest.RequestInfoFrom(ctx)
+	if !found {
+		klog.Error("failed to get requestInfo from context")
+		return nil, false, errors.New("failed to get requestInfo from context")
+	}
```

Comment:
Use `getRequestInfoFromContext` which follows the same pattern as the KMS transformer of not failing the request (because we want the server to keep working where possible).